finetuner/tuner/evaluation.py

```diff
@@ -17,88 +18,104 @@
 from ..embedding import embed
 
 if TYPE_CHECKING:
-    from ..helper import AnyDNN, CollateFnType, DocumentSequence, PreprocFnType
+    from ..helper import AnyDNN, DocumentSequence
 
 
 METRICS = {
-    "r_precision": r_precision,
-    "precision": precision_at_k,
-    "recall": recall_at_k,
-    "f1score": f1_score_at_k,
-    "average_precision": average_precision,
-    "hit": hit_at_k,
-    "reciprocal_rank": reciprocal_rank,
-    "dcg": dcg_at_k,
-    "ndcg": ndcg_at_k,
+    'r_precision': r_precision,
```

Comment:
One thing here - wasn't it also planned to not rely on functions from `docarray.math.evaluation` and instead use methods of `DocumentArray`?

Reply:
yes indeeed, but it's blocked by a change that needs to be done in docarray. I opened a separate issue for this

Reply:
Okay, got it